.travis.yml

```diff
@@ -46,3 +49,9 @@ jobs:
         - docker push "${DOCKER_REPO_NAME}:${DOCKER_TAG}"
         - if [ "$TRAVIS_BRANCH" = "dev" ]; then docker push "${DOCKER_REPO_NAME}:latest"; fi
         - rm ${HOME}/.docker/config.json  # delete token
+
+after_success:
```

Comment:
Not sure about `after_success`, will need to think about that. Might be better suited in `deploy`.

Reply:
Kept in `after_success` in accordance with package documentation on [configuring Travis CI](https://python-semantic-release.readthedocs.io/en/latest/automatic-releases/travis.html?highlight=travis)